app/assets/stylesheets/cookies.scss

```diff
@@ -4,22 +4,48 @@
 
 .cookies-consent {
   position: fixed;
-  bottom: 40px;
-  left: 40px;
-  width: 80%;
-  max-width: 550px;
+  bottom: 0;
+  left: 0;
```

Comment:
@Ayon95 By adding the CSS here, wouldn't it be difficult for you to override from the client-side? Ideally we should have all the CSS for the default cookie consent in the migration. 

CC @Pralish for his thoughts

Reply:
Should I specify a `style` attribute for the elements and move their styles there? The styles in media queries cannot be included in the `style` attribute.

Reply:
@Ayon95 come to think of it, when you customize this you will use your own classes and markup right? In that case you can leave the styles here. 

just remove the `<div class="cookies-consent__overlay"></div>` and implement the background a different way without changing `app/views/cookies/index.html.erb`